composables/usePaymentMethods .ts

```diff
@@ -9,6 +9,7 @@ export const usePaymentMethods = () => {
   const fetcher = nuxt.nuxt2Context.$gqlFetch
   const pciHost = nuxt.nuxt2Context.$config.pciHost
   const apiHost = nuxt.nuxt2Context.$config.apiHost
+  const checkoutOrder = useState<Maybe<Order>>(`use-order-result`, () => null)
```

Comment:
we don't want to duplicate / mutate our checkout state object outside of the useCheckout component

A better approach would be to call the "useCheckout" load method from the component after the add payment method completes

Reply:
The call to "load" looks good now, the checkoutOrder state object is still here in the usePaymentMethods composable though

Reply:
Oops, I missed to remove it. Done